modules/installation-creating-aws-bootstrap.adoc

```diff
@@ -0,0 +1,182 @@
+your cluster initializes correctly.
+====
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in AWS.
+* Create and configure DNS, load balancers, and listeners in AWS.
+* Create control plane and worker roles.
+
+.Procedure
+
+. Create an S3 bucket in your cluster's region and upload the `bootstrap.ign`
+ignition config file to it. This file is located in your installation directory.
+
+.. Create the bucket:
++
+----
+$ aws s3 mb s3://<cluster-name>-upi <1>
+----
+<1> `<cluster-name>-upi` is the bucket name.
+
+.. Upload the `bootstrap.ign` ignition config file to the bucket:
++
+----
+$ aws s3 cp bootstrap.ign s3://<cluster-name>-upi/bootstrap.ign
+----
+
+.. Verify that the file uploaded:
++
+----
+$ aws s3 ls s3://<cluster-name>-upi/
+
+2019-04-03 16:15:16     314878 bootstrap.ign
+----
+
+. Create a YAML file that contains a copy of the CloudFormation template that
+describes the networking objects and load balancers and save it to your
+computer.
+
+. Create a JSON file that contains the parameter values that the template
+requires:
++
+[source,json]
+----
+[
+  {
+    "ParameterKey": "ClusterName", <1>
+    "ParameterValue": "mycluster" <2>
+  },
+  {
+    "ParameterKey": "RhcosAmi", <3>
+    "ParameterValue": "ami-07f604c5f18a1e7a9" <4>
+  },
+  {
+    "ParameterKey": "AllowedBootstrapSshCidr", <5>
+    "ParameterValue": "0.0.0.0/0" <6>
+  },
+  {
+    "ParameterKey": "PublicSubnet", <7>
+    "ParameterValue": "subnet-09c2adf329727bc59" <8>
+  },
+  {
+    "ParameterKey": "MasterSecurityGroupId", <9>
+    "ParameterValue": "sg-0a53cd3337c15a2ee" <10>
+  },
+  {
+    "ParameterKey": "VpcId", <11>
+    "ParameterValue": "vpc-055h956b1e6572be9" <12>
+  },
+  {
+    "ParameterKey": "BootstrapIgnitionLocation", <13>
+    "ParameterValue": "s3://<bucket_name>/bootstrap.ign" <14>
+  },
+  {
+    "ParameterKey": "AutoRegisterELB", <15>
+    "ParameterValue": "yes" <16>
+  },
+  {
+    "ParameterKey": "RegisterNlbIpTargetsLambdaArn", <17>
+    "ParameterValue": "arn:aws:lambda:<zone>:269733483066:function:<dns_stack>-RegisterNlbIpTargets-NHEKB9IHN5XZ" <18>
+  },
+  {
+    "ParameterKey": "ExternalApiTargetGroupArn", <19>
+    "ParameterValue": "arn:aws:elasticloadbalancing:<zone>:269733383066:targetgroup/<dns_stack>-Exter-D374J8IQ3K03/b26760f600ff322e" <20>
+  },
+  {
+    "ParameterKey": "InternalApiTargetGroupArn", <21>
+    "ParameterValue": "arn:aws:elasticloadbalancing:<zone>:269733383066:targetgroup/<dns_stack>-Inter-1Q3IAOSC8ZNNM/8e85701acf15632d" <22>
+  },
+  {
+    "ParameterKey": "InternalServiceTargetGroupArn", <23>
+    "ParameterValue": "arn:aws:elasticloadbalancing:<zone>:269733383066:targetgroup/<dns_stack>-Inter-1INRIWR8L3627/4cd9e764bb418f47" <24>
+  }
+]
+
+----
+<1> A short, representative cluster name to use for hostnames, etc.
+<2> Specify the cluster name that you used when you generated the
+`install-config.yaml` file.
```

Comment:
cluster name is also used when user decide to generate only ignition-configs files.